portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -48,6 +54,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Respond with the result.
     response.setContentType("text/html;");
     response.getWriter().println("Last Sunday, I was " + verb+ " and I saw this " + adj + " " + animal +", who was also " + verb +".");
+    
+    // Create an entity and set its properties 
+    Entity taskEntity = new Entity("Task");
+    taskEntity.setProperty("animal", animal);
+    taskEntity.setProperty("verb", verb);
+    taskEntity.setProperty("adj", adj);
+
+    // Create an instance of DatastoreService class
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(taskEntity);
+ 
+    // Create a query instance
+    Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
+    PreparedQuery results = datastore.prepare(query);
+    for (Entity entity : results.asIterable()) {
+    }
```

Comment:
It doesn't look like we use this query for anything. Can we remove this block of code?

Reply:
Yes! My bad, the tutorial mentioned to add it but I didn't realize it had no function in my code.